recipes/gdb/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "gdb" %}
+{% set version = "7.11" %}
+{% set sha256 = "7a434116cb630d77bb40776e8f5d3937bed11dea56bafebb4d2bc5dd389fe5c1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: gdb-{{ version }}.tar.xz
+  url: http://ftp.gnu.org/gnu/gdb/gdb-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+
+requirements:
+  build:
+    - python
+    - ncurses ==5.9
+    - texinfo
+    - toolchain
+    - xz 5.2.*
+  run:
+    - python
+    - ncurses >=5.9,<6.0
+    - xz >=5.2,<6.0
```

Comment:
Yeah, you could probably do the same for ```xz``` and ```ncurses```. Although for ncurses it will not make much difference as a 5.10 version seems very unlikely.